orchagent/portsorch.cpp

```diff
@@ -479,7 +480,8 @@ bool PortsOrch::getPort(sai_object_id_t id, Port &port)
             }
             break;
         case Port::VLAN:
-            if (portIter.second.m_vlan_info.vlan_oid == id)
+            if (portIter.second.m_vlan_info.vlan_oid == id ||
```

Comment:
I don't see a new `getPort `usage in this PR. So where is it called with `rif_id` as `id`? I think this change may not be required

Reply:
getPort() is called in getAclBindPortId() to get the port information.

Reply:
`getAclBindPortId()` invokes `getPort(alias, port)` with `alias`. Its not this function.